### PR TITLE
Add support for connection avatar icons

### PR DESCRIPTION
**Pull request by bkegley**

### CODE REVIEW

File: src/utils/component/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ComponentDefinition as ComponentDefinitionTemplate } from "@prismatic-io/spectral";
+import { Component as ComponentDefinitionTemplate } from "@prismatic-io/spectral/dist/serverTypes";
```
Comment (bkegley) [inline src/utils/component/index.ts:1]: This was errantly referencing the `ComponentDefinition` prior to running through the convert layer. `prism` handles the result of spectral's `convertComponent`/`convertIntegration`.

File: src/commands/components/publish.ts [hunk 2]
```diff
@@ -110,16 +110,17 @@ export default class PublishCommand extends PrismaticBaseCommand {
 
     const { iconUploadUrl, packageUploadUrl, connectionIconUploadUrls, versionNumber } =
       await publishDefinition(definition, {
-        comment,
-        customer,
```
Comment (pattra) [inline src/commands/components/publish.ts:114]: Why are these being removed?

**Review by pattra — COMMENTED** [review]
overall lgtm, just one small question

File: src/commands/components/publish.ts (hunk 2 above)
Comment (bkegley) [reply src/commands/components/publish.ts:114]: That was a mistake. Good catch

**Review by pattra — APPROVED** [review]

**Pull request merged**